services/sn_analysis_service/app.py

```diff
@@ -28,33 +31,51 @@
 default_analysis_parameters = {"source": "nugent-sn2p", "fix_z": False}
 
 
-def upload_analysis_results(results, data_dict):
+def upload_analysis_results(results, data_dict, request_timeout=60):
     """
     Upload the results to the webhook.
-
-    TO BE COMPLETED
     """
-    log("upload results to webhook")
-    assert isinstance(results, dict)
-    assert isinstance(data_dict, dict)
-    pass
+
+    log("Uploading results to webhook")
+    if data_dict["callback_method"] != "POST":
+        log("Callback URL is not a POST URL. Skipping.")
+        return
+    url = data_dict["callback_url"]
+    try:
+        result = requests.post(
+            url,
+            json=results,
+            timeout=request_timeout,
+        )
+    except requests.exceptions.Timeout:
```

Comment:
If we timeout here then it's precisely because we cannot write back to the SkyPortal instance. So returning something doesn't make sense in this case.